server/server.go

```diff
@@ -1,15 +1,39 @@
 package main
 
 import (
+	"net/http"
+
 	"github.com/labstack/echo/v4"
 )
 
+type MetaStruct struct {
+	Total int `json:"total"`
+}
```

Comment:
This looks great.  I think we can tailor this `struct` a bit more to fit our needs.  For example this isn't just a meta struct (and doesn't really need struct in the name given that it's of type `struct`), it's more akin to a `ContentResponseMeta`, as it's the struct that describes the shape of the `meta` key in the content response.

Reply:
That makes sense, I'll clean that up now. Should I also be organizing this into a separate file for any reason? Not sure what the organization convention here is

Reply:
I don't think we need to do that quite yet, but that'll happen soon.